recipes/README.md

```diff
@@ -96,3 +96,34 @@ command:
 
 For more information, please refer to [this issue](
 https://github.com/71/dance/issues/74#issuecomment-819557435).
+
+## Using `#` to search for current word under cursor
+
+We will use `dance.run` and combine multiple commands to mimic the behavior of `#` in Vim:
+1. Select to the end of the current word
+2. Select back to the begin of the current word
+3. Call search current selection command
+
+```json
+{
```

Comment:
Indentation below is off: indent levels should be 0,2,4,6,..., but are 0,3,7,11,...